src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageControl.xaml.cs

```diff
@@ -48,21 +48,20 @@ private void OnClearEnvironmentVariableGridError(object sender, EventArgs e)
             ClearGridError(dataGridEnvironmentVariables);
         }
 
-        private void OnFocusEnvironmentVariableGridRow(object sender, EventArgs e)
+        private async void OnFocusEnvironmentVariableGridRow(object sender, EventArgs e)
```

Comment:
> this event handler would never throw because BeginInvoke would swallow all exceptions (I believe). 

The code below will crash a WPF app 100% of the time as there is no way to catch this exception.
```C#
private void Button_Click(object sender, RoutedEventArgs e)
{
    Dispatcher.BeginInvoke(new DispatcherOperationCallback((param) =>
    {
        throw new Exception();
    }), DispatcherPriority.Background, new object[] { null });
}
```

In terms of crashing behavior the new code has identical behavior to the old code.